lib/index.js

```diff
@@ -165,6 +165,7 @@ const exec = async (opts) => {
           const prompt = `Need to install the following packages:\n${
           addList
         }Ok to proceed? `
+          typeof log.clearProgress === 'function' && log.clearProgress()
```

Comment:
```suggestion
          if (typeof log.clearProgress === 'function')
            log.clearProgress()
```

The explicit `if` is preferable here for readability